payload/commit.go

```diff
@@ -1,6 +1,8 @@
 package payload
 
-import "github.com/nspcc-dev/neo-go/pkg/io"
+import (
+	"encoding/gob"
+)
 
 // Commit is an interface for dBFT Commit message.
 type Commit interface {
```

Comment:
I'm thinking about moving these into `dbft` as well, but not now.

Reply:
Ref. https://github.com/nspcc-dev/dbft/issues/90.